scripts/start.sh

```diff
@@ -14,6 +14,20 @@ export REPOSITORY=${REPOSITORY:-confluentinc}
 # and expects user to build and provide a local file confluentinc-kafka-connect-replicator-${CONNECTOR_VERSION}.zip
 export CONNECTOR_VERSION=${CONNECTOR_VERSION:-$CONFLUENT}
 
+# Control Center and ksqlDB server must both be HTTP or both be HTTPS; mixed modes are not supported
+# C3_KSQLDB_HTTPS=false: set Control Center and ksqlDB server to use HTTP (default)
+# C3_KSQLDB_HTTPS=true : set Control Center and ksqlDB server to use HTTPS
+C3_KSQLDB_HTTPS=${C3_KSQLDB_HTTPS:-false}
+if [[ "$C3_KSQLDB_HTTPS" == "false" ]]; then
+  export CONTROL_CENTER_KSQL_WIKIPEDIA_URL="http://ksqldb-server:8088"
+  export CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL="http://localhost:8088"
+  C3URL=http://localhost:9021
```

Comment:
All other `URL` variables are preceded by an underscore, consider `C3_URL` here.

Reply:
Ack and fixed